rpcd/playbooks/roles/rpc_support/tasks/retrieve_auth_key.yml

```diff
@@ -13,6 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: Install pip packages for memecache module to work
```

Comment:
This still says "memecache". And while I fully support Internet Memes, I don't think this is the appropriate place to cache them.